CHANGES.md

```diff
@@ -1,102 +1,105 @@
-Changelog
-=========
+# Changelog
 
-DEV
---------------------
-* [#56](https://github.com/GlobalFishingWatch/pipe-tools/issues/56)
-  Option to generate standard SQL in bigquery QuerHelper
-  
-1.0.0 - (2018-11-21)
---------------------
+All notable changes to this project will be documented in this file.
 
-* [#50](https://github.com/GlobalFishingWatch/pipe-tools/pull/50)
-  * Pin Airflow version to 1.10.1
-  * NOTE: This changes where not tested on existing pipelines with 1.9.0
+The format is based on [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-0.2.4 - (2018-11-15)
---------------------
+## [Unreleased]
 
```

Comment:
Add an entry here stating that we are removing all the airflow code and moving to the new repo. Include a link to the new repo.

Reply:
@enriquetuya good idea! Thanks!